eng/pipelines/runtime-official.yml

```diff
@@ -30,6 +30,7 @@ variables:
   value: .NETCore
 - name: _DotNetValidationArtifactsCategory
   value: .NETCoreValidation
+- group: DotNet-Runtime-SDLValidation-Params
```

Comment:
Why is this being checked into our official builds? This will impact both our official build performance and reliability. This should be done at PR time so that the violations are caught before the change is merged, not caught at official build time.

Reply:
We can't do this in PR builds, this needs internal resources, this is how all the SDL validation works. 

cc: @markwilkie @sunandabalu

Reply:
Yes I'm digging into that presently.